plugins/org.python.pydev.debug/src_console/org/python/pydev/debug/newconsole/PydevConsoleConstants.java

```diff
@@ -85,6 +89,29 @@
             { "OpenGL (glut)", "glut" },
             { "pyglet (pyglet)", "pyglet" } };
 
+    public static final String ACTIVE_EDITOR_INTERPRETER_REPRESENTATION = "active editor";
+    public static final String PYTHON_INTERPRETER_REPRESENTATION = "python";
+    public static final String PYDEV_INTERPRETER_REPRESENTATION = "pydev";
```

Comment:
Please rename this to `PYDEV_DEBUG_INTERPRETER_REPRESENTATION = "pydev_debug"` (as this is only valid as the console for a debug session).